test/upgradeability/AdminUpgradeabilityProxy.test.js

```diff
@@ -295,34 +295,34 @@ contract('OwnedUpgradeabilityProxy', ([_, owner, anotherAccount]) => {
       assert.equal(implementation, this.implementation_v0);
     })
 
-    it('should store the owner proxy in specified location', async function () {
-      const position = web3.sha3("org.zeppelinos.proxy.owner");
-      const proxyOwner = await web3.eth.getStorageAt(this.proxyAddress, position);
+    it('should store the admin proxy in specified location', async function () {
+      const position = web3.sha3("org.zeppelinos.proxy.admin");
+      const admin = await web3.eth.getStorageAt(this.proxyAddress, position);
```

Comment:
use `proxyAdmin` here, otherwise the assertion bellow will always be true

Reply:
@maraoz Can you run a linter on the tests to make sure there aren't other cases of variable shadowing?

Reply:
I'm using jshint with the ['shadow' option](http://jshint.com/docs/options/#shadow) and it's not detecting it :(